src/schema/users.ts

```diff
@@ -508,6 +514,12 @@ export const typeDefs = /* GraphQL */ `
     tags: [TagsReadingStatus]
   }
 
+  type StreakRecoverQuery {
+    canDo: Boolean!
```

Comment:
Don't want to spool the whole change, but maybe something like `canRecover` would have been more clear to what it actually means.

No action needed.

Reply:
I'm actually pro to change name, we had a similar thread with @nensidosari and if keep raising this change need to be renamed. `canRecover` is okay for both you?

Reply:
Works for me :)

Reply:
Works for me as well